generated/nifake/nifake/unit_tests/test_grpc.py

```diff
@@ -721,7 +721,7 @@ def test_get_attribute_int32(self):
         response_object = self._set_side_effect(library_func, attribute_value=test_number)
         interpreter = self._get_initialized_stub_interpreter()
         attr_int = interpreter.get_attribute_vi_int32('', attribute_id)
-        assert(attr_int == test_number)
+        assert (attr_int == test_number)
```

Comment:
>get rid of all those redundant parenthesis

You'll probably regret this request when you see how many there are, but I'll do it. Have to be careful, because they aren't always redundant. Sometime it's a multi-line assert or the parentheses are for a tuple.